### ISSUE TITLE
Feature suggestion: no_skip_blank option

### ISSUE THREAD

**Issue by aroberge** [alexmojaki/stack_data#35]
Currently, and as mentioned in the README, `stack_data` design includes a smart choice **for most users** in that it does not waste vertical space by including blank lines. Here's an example of such an output generated by `stack_data`:
```
       5 | cols = []
       6 | rows = []
       8 | for i in range(3):
-->    9 |     rows.appendh(i)
               ^^^^^^^^^^^^
      10 |     print(rows)
```
Looking at the line numbers, one can see that line 7 was a blank line.

I would argue that, **for beginners**, the absence of a blank line in the output makes it more difficult for them to quickly identify the part of their code that was problematic.

`friendly_traceback` made a different choice: single blank lines are included in the output as shown below:
```
        5: cols = []
        6: rows = []
        7:
        8: for i in range(3):
    --> 9:     rows.appendh(i)
               ^^^^^^^^^^^^
       10:     print(rows)
```
When more than one blank lines are included in the original code, they are skipped in friendly_traceback's output but with a different marker, allowing for easy identification that the content was skipped:
```
        5: cols = []
        6: rows = []
        |
        9: for i in range(3):
    -->10:     rows.appendh(i)
               ^^^^^^^^^^^^
       11:     print(rows)
```

While I believe that `stack_data`'s current design choice is correct for most users, I think that it would be useful if one could specify an option so that single blank lines could be included like they are in `friendly-tracback`. This is what it would look like when a single blank line is present with that option specified:
```
       5 | cols = []
       6 | rows = []
       7 |
       8 | for i in range(3):
-->    9 |     rows.appendh(i)
               ^^^^^^^^^^^^
      10 |     print(rows)
```

In addition, it might perhaps be even more useful if the option could also be used to identify when multiple blank lines are present. For example, here's a fictitious output that adds a colon instead of line numbers when more than one blank lines are skipped.

```
       4 | cols = []
       5 | rows = []
       : |
       8 | for i in range(3):
-->    9 |     rows.appendh(i)
               ^^^^^^^^^^^^
      10 |     print(rows)
```

If this was deemed desirable, I could try to implement this **optional** feature in `stack_data`.

**Comment by alexmojaki** [comment]
Yes, I would like to have this as an option.

**Comment by aroberge** [comment]
Ok, I will have a look at it. It might take a few days before I have something working.

Currently, my fork highlights differently than yours when the highlighted code spans multiple lines as it treats the first highlighted line differently in **all cases**, and not only when the highlighted region of the first line is to the right (i.e. higher column number) than the highlighted block below like your version does.  I did change the content of one of the "golden files" in my fork to reflect this change, and all tests should be passing. (See https://github.com/alexmojaki/stack_data/compare/master...aroberge:stack_data:master)

Should I use my current fork as the starting point, or should I remove the changes I made and start from your current master?
